js/plugins/verticalLine.js

```diff
@@ -1,68 +1,111 @@
-/*
- * Handles the drawing of a vertical line
- * on hover over the timeline in the simple view
+/**
+ * Plugin to handle the drawing of a vertical line 
+ * on hovering over the timeline
+ * @type {clinicalTimelinePlugin}
  */
-var clinicalTimelineVerticalLine = function(enableVerticalLine, beginning, ending, advancedView, tooltipOnVerticalLine) {
-  if (enableVerticalLine) {
-    var hoverLineGroup = d3.select(".timeline").append("g")
-      .attr("class", "hover-line");
-  
-    var hoverLine = hoverLineGroup
-      .append("line")
-      .attr("x1", 0)
-      .attr("x2", 0) 
-      .attr("y1", 20)
-      .attr("y2", 268)
-      .style("stroke", "#ccc");
+function clinicalTimelineVerticalLine(name, spec){
+  clinicalTimelinePlugin.call(this, name, spec);
+  this.id = "verticalLine";
```

Comment:
id is during generating the elements in configButton (https://github.com/cBioPortal/clinical-timeline/pull/98/files#diff-2fee492887685c3cdd593e7b14be685dR19) and then to enable/disable the plugins (https://github.com/cBioPortal/clinical-timeline/pull/98/files#diff-2fee492887685c3cdd593e7b14be685dR35)